untangle-suricata-config/files/etc/logrotate.d/suricata.log

```diff
@@ -1,9 +0,0 @@
-/var/log/suricata/*.log {
-	rotate 2
```

Comment:
talk to @sdelafond about seeing if we need a postinst script to fixup the suricata log